### ISSUE TITLE
Crash upon opening a particular chest

### ISSUE THREAD

**Issue by YellowAfterlife** [TurningWheel/Barony#57]
This is on the current beta branch build.

Here's savegame and log file (which seems to log the level seed):
[Barony.zip](https://github.com/TurningWheel/Barony/files/932305/Barony.zip)
The chest in question is the one over that small abyss\gap in a building.

If there's anything else that should be included, do tell. Seems kind of occasional, considering that I've seen it thrice in 50 hours played.

**Comment by addictgamer** [comment]
I wasn't able to reproduce this with the provided save.

The beta fixed a crash when closing chests, but you've experienced several crashes when opening chests?

**Comment by YellowAfterlife** [comment]
Indeed - each of these times the game crashed the moment a chest is opened. The fact is also the last thing to appear in the log. Would running from source code generally bring up the debugger in such cases instead of hard crashing?

**Comment by addictgamer** [comment]
No, not in and of itself. You'd need to compile the code with debugging symbols and run it via a debugger.
You're on Windows, right? If I recall correctly, in Visual Studio, this is typically done via building a debug build and then using the "debug" option to launch Visual Studio's debugger.

**Comment by addictgamer** [comment]
If you're interested in running a debug build, I could look into getting a binary built with debug symbols and shipping that off to you, so that you might not have to deal with setting up the development environment?
You'd still need to somehow launch it in a debugger, and I don't know how one would do that in Visual Studio without also first compiling the code.
